cmd/thanos/compact.go

```diff
@@ -689,6 +693,27 @@ func runCompact(
 	return nil
 }
 
+func validateFilterConf(filterConf *store.FilterConfig) error {
+	if filterConf.MinTime.PrometheusTimestamp() > filterConf.MaxTime.PrometheusTimestamp() {
```

Comment:
Is it valid to have them equal? If the compactor `MinTime` and `MaxTime` are equal it will just be non-functional iirc.